src/site/content/en/blog/generic-sensor/index.md

```diff
@@ -0,0 +1,478 @@
+
+More information on browser implementation status can be found on
+[chromestatus.com](https://www.chromestatus.com/features/5698781827825664?embed)
+
+## What are all these sensors? How can I use them? {: #what-are-sensors-how-to-use-them }
+
+Sensors is a quite specific area which might need a brief introduction. If you are familiar with
+sensors, you can jump right to the [hands-on coding section](#lets-code). Otherwise, let's look at
+each supported sensor in detail.
+
+### Accelerometer and linear acceleration sensor {: #acceleration-and-linear-accelerometer-sensor }
+
+<div class="w-figure w-figure--inline-right" style="max-width:50%">
+  <figure>
+    <img  src="accelerometer.gif"
+         alt="Accelerometer sensor measurements">
+    <figcaption class="w-figcaption"><b>Figure 1</b>: Accelerometer sensor measurements</figcaption>
+  </figure>
+</div>
+
+The [`Accelerometer`](https://www.w3.org/TR/accelerometer/#intro) sensor measures
+acceleration of a device hosting the sensor on three axes (X, Y, and Z). This
+sensor is an inertial sensor, meaning that when the device is in linear free
+fall, the total measured acceleration would be 0 m/s<sup>2</sup>, and when a
+device lying flat on a table, the acceleration in upwards direction (Z axis)
+will be equal to the Earth's gravity, i.e. g ≈ +9.8 m/s<sup>2</sup> as it is
+measuring the force of the table pushing the device upwards. If you push the
+device to the right, acceleration on the X axis would be positive, or negative
+if the device is accelerated from the right toward the left.
+
+Accelerometers can be used for things like: step counting, motion sensing, or
+simple device orientation. Quite often, accelerometer measurements are combined
+with data from other sources in order to create fusion sensors, such as,
+orientation sensors.
+
+The [`LinearAccelerationSensor`](https://w3c.github.io/accelerometer/#linearaccelerationsensor-interface)
+measures acceleration that is applied to the device hosting the sensor,
+excluding the contribution of a gravity force. When a device is at rest, for
+instance, lying flat on the table, the sensor would measure ≈ 0 m/s<sup>2</sup>
+acceleration on three axes.
+
+### Gravity sensor {: #gravity-sensor }
+
+It is already possible for users to manually derive readings close to those of a gravity sensor
+by manually inspecting `Accelerometer` and `LinearAccelerometer` readings, but this can be
+cumbersome and depend on the accuracy of the values provided by those sensors.
+Platforms such as Android can provide gravity readings as part of the operating system,
+which should be cheaper in terms of computation, provide more accurate values depending
+on the user's hardware, and be easier to use in terms of API ergonomics. The
+[`GravitySensor`](https://w3c.github.io/accelerometer/#gravitysensor-interface) returns
+the effect of acceleration along the device's X, Y, and Z axis due to gravity.
+
```

Comment:
```suggestion

As of February 2021, GravitySensor is still under development with no estimated release date.

```

Reply:
Will link to https://bugs.chromium.org/p/chromium/issues/detail?id=1163993 to make this clearer.